features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsNode.kt

```diff
@@ -129,9 +130,10 @@ class RoomDetailsNode @AssistedInject constructor(
     override fun View(modifier: Modifier) {
         val context = LocalContext.current
         val state = presenter.present()
+        val coroutineScope = rememberCoroutineScope()
 
         fun onShareRoom() {
-            this.onShareRoom(context)
+            coroutineScope.onShareRoom(context)
```

Comment:
use scope of the node instead of the view?

Reply:
Ah yes, done